pkg/apiserver/apiserver_test.go

```diff
@@ -2699,3 +2700,23 @@ func TestUpdateChecksAPIVersion(t *testing.T) {
 		t.Errorf("unexpected response: %s", string(b))
 	}
 }
+
+func TestEncodeStatus(t *testing.T) {
+	status := &unversioned.Status{
+		Status:  unversioned.StatusFailure,
+		Code:    200,
+		Reason:  unversioned.StatusReasonUnknown,
+		Message: "",
+	}
+	v1Codec := testapi.Default.Codec()
+	expCodec := testapi.Experimental.Codec()
+
+	_, err := v1Codec.Encode(status)
+	if err != nil {
+		t.Errorf("unexpected error: %v", err)
+	}
+	_, err = expCodec.Encode(status)
+	if err != nil {
+		t.Errorf("unexpected error: %v", err)
+	}
```

Comment:
Make an assertion about the output; the v1Codec should have an apiVersion of "v1" and the experimental one should have a blank apiVersion, right?